docs/libraries.md

```diff
@@ -1,52 +1,144 @@
-[languages]: https://wiki.gnome.org/Projects/GObjectIntrospection/Users
-[GObjectIntrospection]: https://wiki.gnome.org/Projects/GObjectIntrospection
-[brickman]: https://github.com/ev3dev/brickman
-
-## Extra languages
-We also have many great contrubutors that are maintaining extra libraries for
-languages not included in our other repository.  Note that some of these
-libraries may be outdated due to the fast development pace of ev3dev.
-
-* Extra languages:
-    * [Go](https://github.com/ldmberman/GoEV3) updated for ev3dev-jessie by @ldmberman, [original](https://github.com/mattrajca/GoEV3) by @mattrajca
-    * [Go](https://github.com/ev3go/ev3dev) closely following the ev3dev API specification by the @ev3go project.
-    * [Java](https://github.com/ev3dev-lang-java/ev3dev-lang-java) an interface similar to that of LeJOS by @jabrena
-    * [Python](https://github.com/topikachu/python-ev3) by @topikachu
-    * [C (with optional Perl, Python and Ruby bindings)](https://github.com/in4lio/ev3dev-c) by @in4lio
-    * [C](https://github.com/theZiz/ev3c) by @theZiz
-    * [Clojure](https://github.com/annapawlicka/clj-ev3dev) by @annapawlicka
+Before you can start writing code that runs on ev3dev, you need to choose a
+programming language. Community members have built libraries that let you use
+featuressuch as motors and sensors from your code. Choose the language you want
+from below and follow the link to the library's homepage get started.
+
+## Python
+<https://github.com/rhempel/ev3dev-lang-python>
+
+Python is a high-level, general-purpose, interpreted scripting language. It is
+great for beginners, while also powerful for experienced coders.If you are new
+to programming and unsure which language to choose, Python is a great choice!
+
+**Warning!** There is another Python library named `python-ev3` created by
+@topikachu which is _not_ the same as this one. When searching for documentation
+online, make sure that you are reading about @rhempel's `ev3dev-lang-python`.
+
+[Get started with Python](https://github.com/rhempel/ev3dev-lang-python){: .btn .btn-default }
+
+## JavaScript
+<https://github.com/wasabifan/ev3dev-lang-js>
+
+JavaScript is the scripting language of the Web. With a program called Node.js,
+you can write JavaScript that can be run locally like any other scripting
+language. JavaScript with Node.js is great for writing web servers and other
+asynchronous programs which don't need to run tight loops or precise timers. It
+has a syntax similar to that of C and related languages, and is relatively easy
+to learn.
+
+[Get started with JavaScript](https://github.com/wasabifan/ev3dev-lang-js){: .btn .btn-default }
+
+## Go
+<https://github.com/ev3go/ev3dev>
+
+Go is a compiled, statically-typed language created at Google. It aims to be
+simple and light while still providing modern language features. While it is a
+compiled language, cross-compilation is _not_ required to run code on ev3dev,
```

Comment:
Cross-compilation is required. A separate cross-compiler is not - it is built in.